### DISCUSSION TITLE
How long it take to get Github Student pack

### DISCUSSION THREAD

**Discussion by Git-yashabi** [community/community#145836] [GitHub Education]
### Hi everyone,

Hi, How long it take to get Github Student pack, It's been 3 days since i was approved

**Comment by AlexLamper** [comment]
They aim for 1-3 days, but sometimes it takes longer. Mine was approved after a week or so, so you'll probably just have to wait a few more days.

Also make sure the following:

- You are currently enrolled as a student at a degree-granting institution, such as a high school, secondary school, college, university, residential school or similar educational institution.
- You have a verifiable university email address or upload documents confirming your current student status.
- You have a personal GitHub account.
- You are at least 13 years old.

**Comment by davevad93** [comment]
Hi @Git-yashabi ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Git-yashabi was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
